packages.yml

```diff
@@ -1,6 +1,3 @@
 packages:
   - package: dbt-labs/dbt_utils
     version: [">=1.0.0", "<2.0.0"]
-
-  - package: calogica/dbt_expectations
-    version: 0.8.5
```

Comment:
We don't actually use this dependency in the main package, and it was pinned to a patch version. Removed to avoid conflict with other dependencies.